src/handlers/payout/post.ts

```diff
@@ -73,6 +75,30 @@ export const incentivizeComments = async () => {
   await addCommentToIssue(comment, issue.number);
 };
 
+const generatePermitForCommit = async (
+  user: string,
+  comments: string[],
+  multiplier: number,
+  commentElementPricing: Record<string, number>,
+  tokenSymbol: string,
+  node_id: string
+): Promise<[string | undefined, string | undefined]> => {
```

Comment:
```suggestion
): Promise<{payoutUrl?: string, amountInETH?: string}> => {
```

I guess this seems more meaningful. You might need to change the relevant lines as well.